docs/cpp/enable-logging-cpp.md

```diff
@@ -28,3 +27,38 @@ To turn on full logging for an issue report, follow these steps:
 1. Select the "C/C++" option in the log filter selector:
 
    ![Log filter selector](images/cpp/log-filter-selector.png)
+
+## Enable logging for the debug adapter
+Enabling logging for the debug adapter will show communication information between VS Code and our extension and between our extension and the debug adapter.
+
+### Logging for `MI` debuggers
+
+The logging block with its defaults is as follows:
```

Comment:
There is no mention of `launch.json` before this section.  Consider mentioning that the logging block should be added to the configurations in `launch.json`